scrapers/infernalrestraints.yml

```diff
@@ -0,0 +1,40 @@
+name: infernalrestraints
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - infernalrestraints.com/device
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //a[@class="articleTitleText"][1]/text()
+      Date:
+        selector: //span[@class="articlePostDateText"][1]/text()
+        postProcess:
+          - parseDate: "January 02, 2006"
+          - parseDate: "January 2, 2006"
+      Details:
+        selector: //td[@class="articleCopyText"]/text()
+        concat: "<br>"
+        postProcess:
+          - replace:
+            - regex: <br>
+              with: "\n"
+      Performers:
+        Name: //a[@class="articleTitleText"][2]/text()
+      Tags:
+        Name:
+          selector: //td[contains(text(),"Tags:")]/text()
+          # To remove the Tags: text from the tags
+          postProcess:
+            - replace:
+              - regex: "Tags:"
+                with:
```

Comment:
The tags seem to have some extra spaces included  after the comma. It doesn't show up in the UI but is shows in the console log if you set it to debug. 
Adding an extra regex after the tags replace fixes that
```yaml
            - replace:
              - regex: "Tags:"
                with:
              - regex: ",\\s*([^,]+)"
                with: ",$1"
```

Reply:
updated